test/shape.test.js

```diff
@@ -1,11 +1,13 @@
 'use strict';
 
+const assert = require('assert').strict;
 const { readFileSync } = require('fs');
 const path = require('path');
 const { DOMParser } = require('@xmldom/xmldom');
-const should = require('should');
 const SVGSpriter = require('../lib/svg-sprite.js');
 
+const isObject = obj => typeof obj === 'object' && !Array.isArray(obj) && obj !== null;
```

Comment:
I think it's better to add [lodash.isObject](https://www.npmjs.com/package/lodash.isobject) to project dependencies

Reply:
I don't think we should keep using lodash anymore and we should in fact try to get rid of it going forward.

Reply:
Well, maybe.
I checked lodash's method and it differs from this. In this case it should stay the same

Reply:
The current utility works for our case. We should keep it as is for now since it doesn't block this PR.

Later we can try to remove more lodash uses :)